zerver/tests/test_import_export.py

```diff
@@ -137,6 +132,10 @@ def setUp(self) -> None:
         super().setUp()
         self.rm_tree(settings.LOCAL_UPLOADS_DIR)
 
+        # The test database has RealmEmoji records that
+        # don't have files stored for them.
+        RealmEmoji.objects.all().delete()
```

Comment:
This comment should perhaps be extended with expressing that this happens because `LOCAL_UPLOADS_DIR` gets rewritten to a different directory when running each test worker process, and those new directories likely don't have the files present in the original test database's uploads directory.  (I guess I just noticed that the directories are only per-process, not per-test, which may not be the right design choice).

`settings.LOCAL_UPLOADS_DIR = get_or_create_dev_uuid_var_path` being the code that resets this during test startup.

(I suppose we could try to write a shared helper method to copy them over as an alternative solution).